Sources/EpoxyCore/SwiftUI/EpoxySwiftUIHostingView.swift

```diff
@@ -183,12 +183,9 @@ public final class EpoxySwiftUIHostingView<RootView: View>: UIView, EpoxyableVie
     // The view controller must be added to the view controller hierarchy to measure its content.
     addViewControllerIfNeededAndReady()
 
-    // As of iOS 15.2, `UIHostingController` now renders updated content asynchronously, and as such
-    // this view will get sized incorrectly with the previous content when reused unless we invoke
-    // this semi-private API. We couldn't find any other method to get the view to resize
-    // synchronously after updating `rootView`, but hopefully this will become a public API soon so
-    // we can remove this call.
-    viewController._render(seconds: 0)
```

Comment:
let's gate this change behind a `CollectionViewConfiguration` flag to be safe

Reply:
Ah are you proposing to add a new parameter as in the `CollectionViewConfiguration` [here](https://sourcegraph.d.musta.ch/airbnb/apps/-/blob/ios/Pods/managed/EpoxyCollectionView/Sources/Sources/EpoxyCollectionView/CollectionView/CollectionViewConfiguration.swift?L16) which has several bool parameters for the collection view? In those cases the collection view directly used the parameters to change conditions but in this case we're modifying not the `CollectionView` in `EpoxyCollectionView` but rather the `EpoxySwiftUIHostingView`. Would you want this to be passed through collection view to this view?

Reply:
Yes, we'll add a new parameter to `CollectionViewConfiguration` and pass it through to `EpoxySwiftUIHostingView` similar to what we did in this PR https://github.com/airbnb/epoxy-ios/pull/138

Reply:
I looked into adding the parameter into `CollectionViewConfiguration` but it turned out to be pretty tricky because we needed to pass it through several layers and then do a cast on the view to see if it's of the type `EpoxySwiftUIHostingView`. I ended up instead adding this as a parameter to the style for `EpoxySwiftUIHostingView` which make more sense because it's only dealing with code in the hosting view and not with the broader collection view. 

My thinking is that if we have other params we want to add in the future for the EpoxySwiftUIHostingView this could be a configuration similar to the `CollectionViewConfiguration`. In practice, though we can probably clean this parameter up once it's seen that there are no negative issues resulting from the new layout path